tests/test_training/test_training_loop.py

```diff
@@ -85,3 +85,10 @@ class SoftPlusLCWATrainingLoopTestCase(cases.TrainingLoopTestCase):
 
     cls = LCWATrainingLoop
     loss_cls = SoftplusLoss
+
+
+class SymmetricLCWATrainingLoopTestCase(cases.TrainingLoopTestCase):
+    """Test for symmetric LCWA with cross-entropy."""
+
+    cls = SymmetricLCWATrainingLoop
+    loss_cls = CrossEntropyLoss
```

Comment:
is there a whitelist/blacklist of losses usable with Symmetric LCWA?

Reply:
All losses which support LCWA should also be able to support SymLCWA. And, afaik, all losses support LCWA.